docs/guide/custom-snipptes.md

```diff
@@ -78,6 +83,97 @@ now the page-object contains our new function.:
   }
 ```
 
+* ### CustomSnippets For FillForm
+
+In this type you can extend the [FillForm](./add-meethodes.md#addfillform) methode, to let you fill out your custom form elements.
```

Comment:
Wo wird es das custom snippet in fillform sein?
Anfang? Mitte? Ende?

Reply:
Gleiches wäre bei clearForm interessant

Reply:
all custom snippets will be inserted sorted by the elements. The order is not relevant...

Reply:
Ok. My thought was just that maybe somebody wants to change the order. For example if the last input field is a field that only triggers validation if you clicked somewhere else. So he would not want to fill it out at the end.
But like Michel said, usually you fill out in order and those are edge cases.
Just wanted to add that.

Reply:
So you could just add:

In the fillForm method the methods to fill the different elements are ordered the same way the elements are ordered in the DOM.

Reply:
that is not necessarily true. The order can be out of order, for example if an element is displayed later.